background/services/enrichment/index.ts

```diff
@@ -144,10 +144,14 @@ export default class EnrichmentService extends BaseService<Events> {
           ),
         }
       } else {
+        const toName = await this.nameService.lookUpName(transaction.to, ETHEREUM, false);
         // Fall back on a standard contract interaction.
         txAnnotation = {
           timestamp: resolvedTime,
           type: "contract-interaction",
+          displayFields: {
+            contractName: toName
+          }
```

Comment:
No need for the extra indirection here imo:

```suggestion
          contractName: toName
```